tests/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Rules/DependencyRuleTests.cs

```diff
@@ -3,13 +3,13 @@
 using System.Xml.Linq;
 using System.Xml.XPath;
 
-namespace Microsoft.VisualStudio.ProjectSystem.Rules
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Rules
 {
     public sealed class DependencyRuleTests : XamlRuleTestBase
     {
         [Theory]
         [MemberData(nameof(GetResolvedDependenciesRules))]
-        public void VisibleEditableResolvedDependenciesMustHaveDataSource(string ruleName, string fullPath)
+        public void VisibleEditableResolvedDependenciesMustHaveDataSource(string ruleName, string fullPath, Type assemblyExporterType)
```

Comment:
`assemblyExporterType` doesn't seem to be used in the test. Is the point to make it clear where the `Rule` is coming from, or is it used in other tests that use the same theory data?

Reply:
The latter. It's been added in many places but is only used in one.

Maybe a better approach would be to introduce a type for all these parameters.